ssh/tailssh/incubator.go

```diff
@@ -344,17 +493,25 @@ const (
 	assertPrivilegesWereDroppedByAttemptingToUnDrop = false
 )
 
-// dropPrivileges contains all the logic for dropping privileges to a different
+// dropPrivileges determines the required user ID and group IDs for this
+// incubator and then calls doDropPrivileges to drop privileges for the current
+// process.
+func dropPrivileges(logf logger.Logf, ia incubatorArgs) error {
+	return doDropPrivileges(logf, ia.uid, ia.gid, ia.gids)
+}
```

Comment:
nit: this wrapper seems redundant now

Reply:
I kept it to avoid complicating the test code.